sdk/identity/azure-identity/src/client_secret_credential.cpp

```diff
@@ -9,6 +9,40 @@
 #include <chrono>
 #include <sstream>
 
+namespace {
+// Assumes !scopes.empty()
+std::string FormatScopes(std::vector<std::string> const& scopes, bool asResource)
```

Comment:
What does `asResource` mean?

Reply:
it is internal function. You can only have single scope as resource, and when you want to build resource, you don't add scope suffix.